rmc/utils/generic.py

```diff
@@ -259,6 +279,7 @@ def filter_context_using_gnomad(
 
     # Optionally also filter context HT using gnomAD coverage
     if filter_context_using_cov:
```

Comment:
I also can't remember why this is separated out from the rest of the function, since it looks like it gets called once and is set to True? so I'll remove it entirely
<img width="1475" alt="image" src="https://user-images.githubusercontent.com/19142091/235931165-c7bfd4ec-8005-44cd-9809-98ba84dd00ff.png">